src/store/actions/repos.actions.js

```diff
@@ -0,0 +1,13 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from 'axios'
+
+
+const fetchRepos = createAsyncThunk(
+    'repos/fetchRepos',
+    async () => {
+        const res = await axios.get('https://api.github.com/user/repos')
```

Comment:
If we abstract the baseUrl and make it an environment variable we will directly use it here.
If for some reason, we or the thrid party (github) are changing the url, we will change it once